foundationdb/examples/class-scheduling.rs

```diff
@@ -201,13 +248,13 @@ async fn signup_trx(trx: &Transaction, student: &str, class: &str) -> Result<()>
     trx.set(&class_key, &pack(&(available_seats - 1)));
     trx.set(&attends_key, &pack(&""));
 
-    Ok(())
+    Ok(true)
 }
 
-async fn signup(db: &Database, student: String, class: String) -> Result<()> {
-    db.transact_boxed_local(
+async fn signup(db: &Database, student: String, class: String) -> Result<bool> {
+    db.transact_boxed(
```

Comment:
Could you remove the transact_boxed and use `db.run`, to make it more like the [java bindings](https://apple.github.io/foundationdb/class-scheduling-java.html#signing-up-for-a-class)?